tools/osbuild-mpp

```diff
@@ -575,7 +581,8 @@ class PkgInfo:
 
         checksum = f"{checksum_type}:{checksum_hex}"
 
-        return cls(checksum, pkg.name, pkg.evr, pkg.arch)
+        return cls(checksum, pkg.name, pkg.evr, pkg.arch, pkg.license,
+                   pkg.summary, pkg.sourcerpm, pkg.buildtime, pkg.vendor)
```

Comment:
In some places it is `source_rpm`, here it is `sourcerpm`. Should this not be consistent?

Reply:
The property is called `sourcerpm` in the `dnf` library but `source_rpm` in `mpp`. This is a conversion between the two :)

Reply:
Ok, thanks for the explanation:)

Reply:
sourcerpm is the word used in dnf/hawkey, I started with source_rpm. I can do the change if you like

Reply:
If @supakeen is happy with existing code I am happy too. I just thought it may be some typo :)

Reply:
I made the change